homeassistant/components/recorder/purge.py

```diff
@@ -112,6 +114,118 @@ def _select_event_state_and_attributes_ids_to_purge(
     return event_ids, state_ids, attributes_ids
 
 
+def _generate_find_attr_lambda(attribute_ids: list[int]) -> StatementLambdaElement:
```

Comment:
This works and its fast, but I need to figure out how to avoid writing it all out